package.json

```diff
@@ -25,7 +25,9 @@
   "main": "dist/vast-client-node.min.js",
   "browser": "dist/vast-client.min.js",
   "scripts": {
+    "prebuild": "rm -rf dist_old && mkdir dist_old && cp -a dist/. dist_old/",
```

Comment:
Not windows (cmd) proof I think but I'm not sure if it's actually a problem

Reply:
I'm not sure how I would make it work on both windows and mac 🤔 is it ok if I leave it as is?

Reply:
I guess it's ok like that, there is still ways for windows users to use it, so install the bash from "windows features" (from  what I quickly checked on the internet) or use Git bash (what I do)